dash/development/build_process.py

```diff
@@ -108,11 +111,11 @@ def digest(self):
 
     @job("copy and generate the bundles")
     def bundles(self, build=None):
-        if not os.path.exists(self.build_folder):
+        if not os.path.exists(self.deps_folder):
```

Comment:
we can use the `exist_ok` flag in `os.makedirs` and do away with the if block altogether.

Reply:
good call -> https://github.com/plotly/dash/pull/1685/commits/f424c8b9769d104e28bcd5da5c9f868bc9e9696b

Reply:
ah right, Py2 🙃 🔫

Reply:
lol. k. I thought if its available for versions<=3.6 then it should be available for versions<2. But that is not the case, sorry for that.